tasks/build/package_scripts/pre_remove.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo -n "Stopping kibana service..."
+if command -v systemctl >/dev/null; then
```

Comment:
It could but it isn't necessary, stopping if it's not already running will still exit 0.  The reason for the check on pre install is trying to stop a service that isn't installed will exit > 0